src/js/facility-locator/containers/VAFacility.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { fetchVAFacilities } from '../actions';
+import FacilityDetailsList from './FacilityDetailsList';
+
+class VAFacility extends Component {
+  componentWillMount() {
+    this.props.fetchVAFacilities(this.props.params.id);
+  }
+
+  renderFacility(facility) {
+    if (facility) {
+      return (
+        <div key={facility.name}>
+          <h1>{facility.name}</h1>
+          <FacilityDetailsList key={facility.id} facility={facility}/>
+        </div>
+      );
+    }
+    return (
+      <div></div>
+    );
+  }
+
+  render() {
+    return (
+      <div>
+        {this.renderFacility(this.props.facility)}
```

Comment:
I don't think that needs to be in its own function. Seems perfectly fine to me to decide whether to render anything in `render` itself.